deepinv/physics/tomography.py

```diff
@@ -139,27 +153,41 @@ def __init__(
     def A(self, x, **kwargs):
         if self.img_width is None:
             self.img_width = x.shape[-1]
-        output = self.radon(x)
+        if self.fan_beam or self.adjoint_via_backprop:
+            output = self.radon(x)
+        else:
+            output = ApplyRadon.apply(x, self.radon, self.iradon, False)
         if self.normalize:
             output = output / x.shape[-1]
         return output
 
     def A_dagger(self, y, **kwargs):
```

Comment:
It would be nice to add a docstring here (I don't know why it wasn't there)
```suggestion
    def A_dagger(self, y, **kwargs):
    r"""
    Computes the filtered back-projection (FBP) of the measurements.
    
    .. warning::

        The filtered back-projection algorithm is not the exact linear pseudo-inverse of the Radon transform, but it is a good approximation that is robust to noise.
        
        :param torch.Tensor y: measurements
        :return torch.Tensor: noisy measurements
    """
```